app/controllers/themes_controller.rb

```diff
@@ -1,27 +1,26 @@
 class ThemesController < ::ApplicationController
   def assets
-    theme_id = params[:id].to_i
+    theme_ids = params[:ids].to_s.split("-").map(&:to_i)
 
-    if params[:id] == "default"
-      theme_id = nil
+    if params[:ids] == "default"
+      theme_ids = [nil]
```

Comment:
I was wondering if we can just use a blank array here. That feels like the better representation of the data here.